Fw/FilePacket/test/ut/Main.cpp

```diff
@@ -11,33 +11,6 @@
 
 namespace Fw {
 
-  // Serialize and deserialize a file packet header
```

Comment:
Why was this test deleted?

Reply:
The header test was using private variables. Since serialization of headers was already tested in the start, data, and end packet tests, I didn't think a standalone header test was particularly useful. If you'd like to keep it around I'm happy to re-write it so that it's not using any private variables.